src/connectors/github_organizations.py

```diff
@@ -0,0 +1,341 @@
+    , value:commits::VARIANT commits
+    , value:head_commit::VARIANT head_commit
+    , value:repository::VARIANT repository
+    , value:organization::VARIANT organization
+    , value:sender::VARIANT sender
+    , value:action::VARCHAR(256) action
+    , value:check_run::VARIANT check_run
+    , value:check_suite::VARIANT check_suite
+    , value:number::'NUMBER(38,0)' number
+    , value:pull_requests::VARIANT pull_request
+    , value:labels_url::VARIANT label
+    , value:requested_teams::VARIANT requested_team
+    , value:ref_type::VARCHAR(256) ref_type
+    , value:master_branch::VARCHAR(256) master_branch
+    , value:description::VARCHAR(256) description
+    , value:pusher_type::VARCHAR(256) pusher_type
+    , value:review_comments::VARIANT review
+    , value:changed_files::VARIANT changes
+    , value:comments_url::VARIANT comment
+    , value:issues_url::VARIANT issue
+    , value:id::NUMBER(38,0) id
+    , value:head_sha::VARCHAR(256) sha
+    , value:name::VARCHAR(256) name
+    , value:target_url::VARCHAR(8192) target_url
+    , value:context:: VARCHAR(256) context
+    , value:state:: VARCHAR(256) state
+    , value:commits_url::VARIANT commit
+    , value:branches_url:VARIANT branches
+    , value:created_at::TIMESTAMP_LTZ(9) created_at
+    , value:updated_at::TIMESTAMP_LTZ(9) updated_at
+    , value:assignees_url::VARIANT assignee
+    , value:releases_url::VARIANT release
+    , value:members_url::VARIANT membership
+    , value:FILL IN ALERT
+    , value:FILL IN SCOPE
+    , value:members:VARIANT member
+    , value:requested_reviewers::VARIANT requested_reviewer
+    , value:requested_teams::VARIANT team
+    , value:starred_url::TIMESTAMP_LTZ(9) starred_at
+    , value:has_pages::VARIANT pages
+    , value:has_project::VARIANT project_card
+    , value:deployments_url::VARIANT deployment_status
+    , value:forks::VARIANT forkee
+    , value:milestones_url::VARIANT milestone
+    , value:keys_url::VARIANT key
+    , value:has_projects::VARIANT project_column
+    , value:statuses::VARCHAR(256) status
+    , value:avatar_url::VARCHAR(256) avatar_url
+FROM data.{base_name}_stream, LATERAL FLATTEN(input => v:configurationItems)
+WHERE ARRAY_SIZE(v:configurationItems) > 0
+'''
+
+    db.create_stream(
+        name=f'data.{base_name}_stream',
+        target=f'data.{base_name}_staging'
+    )
+
+    # IAM change takes 5-15 seconds to take effect
+    sleep(5)
+    db.retry(
+        lambda: db.create_pipe(
+            name=pipe,
+            sql=(
+                f"COPY INTO data.{base_name}_staging(v, filename) "
+                f"FROM (SELECT $1, metadata$filename FROM @data.{base_name}_stage/)"
+            ),
+            replace=True,
+            autoingest=True,
+        ),
+        n=10,
+        sleep_seconds_btw_retry=1
+    )
+
+    db.create_task(name=f'data.{base_name}_TASK', schedule='1 minute',
+                   warehouse=WAREHOUSE, sql=config_ingest_task)
+
+    db.execute(f"ALTER PIPE {pipe} REFRESH")
```

Comment:
refresh can take a long time, we should to like the config and cloudtrail connectors where the refresh command is surfaces to the user if they want to backfill